include/aspect/initial_conditions/interface.h

```diff
@@ -93,6 +94,31 @@ namespace aspect
     };
 
 
+    /**
+     * Create a class inheriting from dealii.Function to translate the
+     * interface call to dealii.Function format
+     */
+    template <int dim>
+    class InterfaceFunction : public dealii::Function<dim, double>
+    {
+      public:
+        InterfaceFunction (const Interface<dim> &interface,
+                           const unsigned int selected_component,
+                           const unsigned int n_components);
```

Comment:
Ok, it looks like I was just overlooking the correct format for the call then. Considering using the other model to allow a single composition interpolation call would be beyond the scope of the this issue.